testing/util.py

```diff
@@ -42,6 +42,16 @@ def cmd_output_mocked_pre_commit_home(*args, **kwargs):
     reason='swift isn\'t installed or can\'t be found',
 )
 
+skipif_cant_run_go = pytest.mark.skipif(
+    parse_shebang.find_executable('go') is None,
+    reason='go isn\'t installed or can\'t be found',
+)
+
+skipif_cant_run_bash = pytest.mark.skipif(
+    parse_shebang.find_executable('bash') is None,
+    reason='bash isn\'t installed or can\'t be found',
+)
```

Comment:
without bash, most of the testsuite is useless -- pre-commit tests lots of the framework functionality with script hooks as they are _fast_.  I don't think I can accept a PR which adds skips for `bash`.  As `bash` is necessarily installed on all the platforms we support (it's necessary for `git` itself to function) this also seems inappropriate.